discovery-provider/alembic/versions/dc7f691adc79_add_aggregate_track_table.py

```diff
@@ -0,0 +1,93 @@
+"""add aggregate track table
+
+Revision ID: dc7f691adc79
+Revises: 8e4dda8255fd
+Create Date: 2022-01-20 18:21:40.504845
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "dc7f691adc79"
+down_revision = "8e4dda8255fd"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    connection = op.get_bind()
+    connection.execute(
+        """
+      begin;
+        DROP INDEX IF EXISTS aggregate_track_idx;
+        DROP MATERIALIZED VIEW aggregate_track CASCADE;
```

Comment:
If it cascades and drops other things, we would have to recreate those too. Also, we're trying to enforce idempotency so a migration should be able to upgrade → overwrite alembic_version to prev version → then upgrade again with no issue. . Similar to this.

https://github.com/AudiusProject/audius-protocol/blob/cc8591b39bd7b87fba2f4afc7bef59bc74df708f/discovery-provider/alembic/versions/df4709b4391b_create_archived_plays_table.py#L23